smt/surrogate_models/neural_net/tests/test_neural_net.py

```diff
@@ -0,0 +1,63 @@
+'''
+Author: Steven Berguin <<steven.berguin@gtri.gatech.edu>>
+
+This package is distributed under New BSD license.
+'''
+
+import unittest
+import smt.surrogate_models.neural_net as nn
+
+
+class Test(unittest.TestCase):
+
+    def test_activation(self):
+        print("testing neural_net.activation")
+        nn.activation.test_activation()
```

Comment:
Test have to contain assertions. What are the inputs of the function under test? What is the expected result?

Reply:
Agreed. However, in hindsight, I don't think unit tests are necessary for this and decided to keep such functions insight the module they pertain to with #pragma no cover.